model/browser/events.yaml

```diff
@@ -0,0 +1,104 @@
+    name: browser.resource_timing
+    brief: >
+      This event describes the timing metrics as provided by PerformanceResourceTiming Performance API.
+      These metrics are related to fetching a resource, such as
+      XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+    note: >
+      This event captures data from the [ResourceTiming](https://www.w3.org/TR/resource-timing/).
+      It is recommended to be captured by using the PerformanceResourceTiming API.
+      If the page starts unloading before the page loads,
+      then the partial data can be captured by calling the [performance.getEntriesByType](https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByType) method.
+    attributes:
+      - ref: url.full
+        requirement_level: required
+        note: >
+          Generally the url doesn't include any query strings or any leading credentials,
+          and it's formatted as scheme://full domain/path only, for example https://user:pwd@www.example.com/path?a=b&c=d
+          would be sent as https://www.example.com/path by both removing any potential PII and reducing the cardinality of the name.
+    body:
+      id: browser.resource_timing
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: fetchStart
+          type: double
+          stability: development
+          brief: >
+            The time immediately before the browser starts to fetch the resource,
+            and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [9.600000000558794]
+        - id: domainLookupStart
+          type: double
+          stability: development
+          brief: >
+            The time immediately before the browser starts the domain name lookup for the resource,
+            and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [100.6000000014901]
+        - id: domainLookupEnd
+          type: double
+          stability: development
+          brief: >
+            The time immediately after the browser finishes the domain name lookup for the resource,
+            and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [100.6000000014906]
+        - id: connectStart
+          type: double
+          stability: development
+          brief: >
+             The time immediately immediately before the user agent starts establishing the connection to the server to retrieve the resource,
+             and it is measured in milliseconds.
+          requirement_level: recommended
+          examples: [476.6000000014901]
+        - id: secureConnectionStart
+          type: double
+          stability: development
+          brief: >
+            The time immediately before immediately before the browser starts the handshake process to secure the current connection,
+            and it is measured in milliseconds.
```

Comment:
(same for others)
```suggestion
            The time immediately before immediately before the browser starts the handshake process to secure the current connection, in milliseconds.
```

Reply:
this is covered by previous comments, so resolved this one.